src/orders/components/OrderSummary/PaymentsSummaryEmptyState.module.css

```diff
@@ -0,0 +1,26 @@
+.emptyState {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 32px 16px;
+  gap: 0;
+}
+
+.iconWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 56px;
+  height: 56px;
+  border-radius: 50%;
+  background-color: var(--macaw-color-background-default2);
+}
+
+.textContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 4px;
+  text-align: center;
+}
```

Comment:
issue: all of these classes could be replaced with `Box` component props, these styles don't use our design system tokens (e.g. `width`, `height`). We should avoid using magic numbers in `px`, since it can easily make our whole design misaligned.

Overrides like `gap: 4px;` can be done like so: `__gap="4px"`